airlock/views/request.py

```diff
@@ -81,6 +81,38 @@ def request_view(request, request_id: str, path: str = ""):
         "request_release_files",
         kwargs={"request_id": request_id},
     )
+
+    if path == "":
+        file_approve_url = None
+        file_reject_url = None
+    else:
```

Comment:
not a big fan of this big indent block, might refactor before asking for review

Reply:
Possible alternative: pass the file relpath as a POST parameter like we do for the `workspace_add_file_to_request` view, so the url is just `/requests/approve/<str:request_id>/`. The template logic already deals with things that are or aren't files (displaying content etc)